Makefile

```diff
@@ -54,6 +54,14 @@ else
 GOBIN=$(shell go env GOBIN)
 endif
 
+# Check whether this is a development or release version
+ifeq (,$(shell echo $(IMAGE_VERSION) | grep -iE '(:latest|SNAPSHOT|dev|BETA[[:digit:]]+)$$'))
```

Comment:
Patch overall makes sense to me. I found this conditional counter-intuitive, since it "feels" like it should be checking if the version ends with `:latest`/`xSNAPSHOT`/`xdev` etc., but in fact it's checking the opposite because it's comparing the result to the empty string. IMO it'd be slightly easier to read if this was `ifneq` and the `PULL_POLICY`s swapped, but YMMV. I'm fine with it as is.

Reply:
Agreed. I've switched this to `ifneq` swapped the branches.